Client/Frontend/Browser/BackForwardTableViewCell.swift

```diff
@@ -28,6 +28,7 @@ class BackForwardTableViewCell: UITableViewCell {
         faviconView.layer.borderColor = UIColor(white: 0, alpha: 0.1).cgColor
         faviconView.layer.masksToBounds = true
         faviconView.contentMode = .center
+        faviconView.translatesAutoresizingMaskIntoConstraints = false
```

Comment:
We have something to set `translatesAutoresizingMaskIntoConstraints = false` automatically for us with https://github.com/mozilla-mobile/firefox-ios/blob/8b0258ec2fe338df8d0128c3be16c58a83c58843/Client/Utils/View.swift#L23

This could be used here on both lazy var `faviconView` and `label` in this file

Reply:
Neat function! I went ahead and added in BackForwardListViewController and BackForwardTableViewCell. Thanks for your review!